Sources/FanMenu.swift

```diff
@@ -329,11 +330,11 @@ class FanMenuScene {
         let node = Group(contents: contents)
         node.opacity = 0.0
         
-        node.onTouchPressed { _ in
-            fanMenu.onItemWillClick?(button)
+        node.onTouchPressed { [weak fanMenu] _ in
```

Comment:
I think here we can use unowned here because node without View - it shouldn't be possible in this case